infrastructure/demo.tfvars

```diff
@@ -1,3 +1,7 @@
 idam_api_url = "https://idam-api.demo.platform.hmcts.net"
 
 ccd_ui_base_url = "https://www.ccd.demo.platform.hmcts.net"
+
+idam_client_id = "fpl_case_service"
```

Comment:
Do you know if the id / redirect_uri can be left here or should be added as secret? @geek-soft any opinion?

Reply:
I am not sure of the answer to that, who would be able to confirm? I guess this extends to our chart yml files as well

Reply:
Neither are secret so I believe it is fine to keep as is